packages/grafana-ui/src/components/TimeSeries/TimeSeries.tsx

```diff
@@ -8,7 +8,7 @@ import { preparePlotConfigBuilder } from './utils';
 import { withTheme2 } from '../../themes/ThemeContext';
 import { PanelContext, PanelContextRoot } from '../PanelChrome/PanelContext';
 
-const propsToDiff: string[] = [];
+const propsToDiff: string[] = ['timeRange'];
```

Comment:
hymmm -- I guess this invalidates the whole graph.  maybe we need `GraphNG` to check timeRange explicitly